core/ui/Actions/new-journal.tid

```diff
@@ -2,10 +2,13 @@ title: $:/core/ui/Actions/new-journal
 tags: $:/tags/Actions
 description: create a new journal tiddler
 
-<$vars journalTitleTemplate={{$:/config/NewJournal/Title}} journalTags={{$:/config/NewJournal/Tags}} journalText={{$:/config/NewJournal/Text}}>
+\define get-tags()
+$(journalTags)$ $(moreTags)$
+\end
+<$vars journalTitleTemplate={{$:/config/NewJournal/Title}} journalTags={{$:/config/NewJournal/Tags}} moreTags={{$:/language/DefaultNewTiddlerTags}} journalText={{$:/config/NewJournal/Text}}>
 <$wikify name="journalTitle" text="""<$macrocall $name="now" format=<<journalTitleTemplate>>/>""">
 <$reveal type="nomatch" state=<<journalTitle>> text="">
-<$action-sendmessage $message="tm-new-tiddler" title=<<journalTitle>> tags=<<journalTags>> text={{{ [<journalTitle>get[]] }}}/>
+<$action-sendmessage $message="tm-new-tiddler" title=<<journalTitle>> tags=<<get-tags>> text={{{ [<journalTitle>get[]] }}}/>
```

Comment:
Rather than using a macro with text subtitution, it might be cleaner to do:

```
tags={{{ [enlist<journalTags>] [enlist<moreTags>] }}}
```

(Not tested!)

Reply:
@Jermolene, it doesn't work your way, it just picks the first tag

Reply:
Gosh perhaps we need a "listify" operator to call `$tw.utils.stringifyList()`...

Reply:
>Gosh perhaps we need a "listify" operator to call $tw.utils.stringifyList()...

Such a filter would be useful

Reply:
I'm away from my main computer until tomorrow but will look at it then if you don't get a chance first...

Reply:
@Jermolene I'll let you go first